### ISSUE TITLE
Support reordering work items

### ISSUE THREAD

**Issue by cschleiden** [cschleiden/vsts-quick-decompose#4]
- Support drag and drop for mouse support, and shortcuts like "Ctrl + Cursor Up/Down".
- Move children when moving a parent

**Comment by brettjacobson** [comment]
Agreed this would be a valuable feature; being able to reorder work items as your creating them allows them to be created in the sequence you want, but you may not decide on the sequence until you finish typing in all the items and can think about them as a whole, before hitting Save.
